int32 clock_value
---
bool success